src/static/css/styles.css

```diff
@@ -39,12 +39,24 @@ a {
     text-decoration: none;
 }
 
-a:hover {
+section a:hover {
     background-color: #fff;
     border-color: #667a7d;
     color: #667a7d;
 }
 
+select {
```

Comment:
Is this style applicable to all `select` elements or only the language switcher? If the latter, I'd suggest merging with the style rule below.